cypress-tests/cypress/e2e/admin/pageBuilder/templates/pageTemplatesSorting.cy.ts

```diff
@@ -0,0 +1,101 @@
+import { customAlphabet } from "nanoid";
+
+context("Page Builder - Template Sorting", () => {
+    const nanoid = customAlphabet("abcdefghijklmnopqrstuvwxyz");
+    const titleString1 = "ABC";
+    const titleString2 = "DEF";
+    const titleString3 = "GHI";
+    const titleString4 = "!#$%&/()=?*";
+
+    const pageTemplateData1 = {
+        title: titleString1,
+        slug: nanoid(6),
+        description: nanoid(6),
+        tags: [],
+        layout: "static",
+        pageCategory: "static"
+    };
+    const pageTemplateData2 = {
+        title: titleString2,
+        slug: nanoid(6),
+        description: nanoid(6),
+        tags: [],
+        layout: "static",
+        pageCategory: "static"
+    };
+    const pageTemplateData3 = {
+        title: titleString3,
+        slug: nanoid(6),
+        description: nanoid(6),
+        tags: [],
+        layout: "static",
+        pageCategory: "static"
+    };
+    const pageTemplateData4 = {
+        title: titleString4,
+        slug: nanoid(6),
+        description: nanoid(6),
+        tags: [],
+        layout: "static",
+        pageCategory: "static"
+    };
+
+    beforeEach(() => {
+        cy.login();
+        cy.pbDeleteAllTemplates();
+        cy.wait(1000);
+        cy.pbCreatePageTemplate(pageTemplateData1);
+        cy.pbCreatePageTemplate(pageTemplateData2);
+        cy.pbCreatePageTemplate(pageTemplateData3);
+        cy.pbCreatePageTemplate(pageTemplateData4);
+    });
+
+    it("Should be able to create templates and then sort them correctly", () => {
+        cy.visit("/page-builder/page-templates");
+
+        cy.findByTestId("default-data-list.filter").click();
+        cy.get(".webiny-ui-select select").select("Newest to oldest");
+        cy.findByTestId("default-data-list").within(() => {
+            cy.get("li .mdc-list-item__text")
+                .first()
+                .each($span => {
+                    cy.wrap($span).invoke("text").should("include", titleString4);
+                });
+        });
```

Comment:
Can be written nicer:

```suggestion
  cy.findByTestId("default-data-list").within(() => {
            cy.get("li")
                .first()
                .within(() => cy.findByText(titleString4).should("exist"));
        });
```